src/db/collections/block.rs

```diff
@@ -20,7 +20,7 @@ use crate::{
 
 /// Chronicle Block record.
 #[derive(Clone, Debug, Serialize, Deserialize)]
-struct BlockDocument {
+pub(crate) struct BlockDocument {
```

Comment:
Why do we need those changes?

Reply:
We don't need them anymore since I moved the balance calc. into the outputs collection. Without it we cannot do lookups from other collections.

Reply:
Let's just change only what's necessary for now, reduces the amount of code to look at when searching for errors.

Reply:
what errors? :trollface: